source/voice_control/troubleshooting.markdown

```diff
@@ -87,6 +87,23 @@ The example below shows common pitfalls when enquiring about the weather. While
      ![Create alias for entity name](/images/assist/assist_create_alias.png)
 5. If you just ask "What is the weather" when you have forecast entities for multiple entities, Assist always returns the data for the place that was first added. Currently, there is no way to change that.
 
+## My voice assistant understands me, processes the command, but I do not get back a voice response
```

Comment:
```suggestion
## I don't get a voice response

My voice assistant understands me and processes the command, but I don't get a voice response.
```
Do you think we can shorten the titles, and provide the details in the intro para? It makes the title easier to read in the side nav. (yes, I know. Some of the other titles are long, too :sweat_smile: )